idd/Energy+.idd.in

```diff
@@ -53026,6 +52989,28 @@ Coil:Cooling:DX:VariableSpeed,
         \type real
         \minimum 0
         \default 0
+   N7,  \field Maximum Cycling Rate
```

Comment:
Does Coil:Cooling:DX:VariableSpeed need a bump on `\min-fields`?

Reply:
It's currently 31, but after the 90.1 metric update the last default field is N93. The prior alpha field is A45...so maybe it should be 93+45=138?

Reply:
Yeah, there's exceptions to the rule, min-fields should also cover the last required-field, and if there are defaulted fields in extensible groups (or pseudo-extensible like this one), min-fields should stop at the last required group, in this case the Speed 1 inputs. So, @rraustad is correct, 36.